### ISSUE TITLE
Crossfade is not working in the Compose version

### ISSUE THREAD

**Issue by rex50** [bumptech/glide#5016]
**Glide Version**: Compose 1.0.0-alpha.1

**Integration libraries**: No

**Device/Android Version**: HTC U11

**Issue details / Repro steps / Use case background**: Cross fade is not working while using with requestbuilder.transition

**Glide load line / `GlideModule` (if any) / list Adapter code (if any)**:
```java
    GlideImage(
        model = source,
        modifier = modifier,
        contentDescription = contentDescription,
        contentScale = ContentScale.Fit,
        colorFilter = colorFilter,
        alpha = 1.0f,
    ) { requestBuilder ->
        requestBuilder.transition(DrawableTransitionOptions().crossFade(DrawableCrossFadeFactory.Builder(1000).setCrossFadeEnabled(true).build()))
    }
```

**Comment by sjudd** [comment]
Animations are not currently supported :/

**Comment by rex50** [comment]
I think it would helpful if we can add this info the docs.

**Comment by EvgenyMamrukov** [comment]
Could somebody tell approximate release date of stable glide compose version (non alpha)?

**Comment by ilansas** [comment]
Is there any plans to add support for animations any time soon ? Any workaround for now ?

**Comment by sjudd** [comment]
Fixed by d10c5ff

**Comment by tamtom** [comment]
yea it was reverted back ! 
https://github.com/bumptech/glide/pull/5514
how can we support crossfade ?
